package.json

```diff
@@ -20,8 +20,8 @@
     "deploy": "npm install && composer install --no-dev -o && gulp",
     "lint-js": "eslint assets/js",
     "format-js": "eslint --fix assets/js",
-    "lint": "npm run lint-js",
-    "format": "npm run format-js"
+    "lint": "npm run lint-js && npm run lint-php",
```

Comment:
@tlovett1 need to update this to remove `npm run lint-php` and `npm run format-php`

Reply:
If you want cross platform, npm run lint-js && npm run lint-php won't work.

Look at https://www.npmjs.com/package/npm-run-all

Reply:
I removed these npm scripts.